newIDE/app/src/Utils/ColorTransformer.js

```diff
@@ -11,3 +11,24 @@ export const rgbToHex = (r: number, g: number, b: number) =>
  */
 export const rgbToHexNumber = (r: number, g: number, b: number) =>
   (r << 16) + (g << 8) + b;
+
+/**
+ * Convert a hex color value to an rgb object value.
+ */
+export const getRGBColorFromHex = (hex: string) => {
+  if (/^#([A-Fa-f0-9]{3}){1,2}$/.test(hex)) {
+    let c = hex.substring(1).split('');
+    if (c.length === 3) {
+      c = [c[0], c[0], c[1], c[1], c[2], c[2]];
+    }
+    const cols = '0x' + c.join('');
+    return {
+      r: parseInt((cols >> 16) & 255),
```

Comment:
You're doing `cols >> 16`, so you're applying ">>" operator on cols which is a string. This does not look correct?

Reply:
I just noticed Pixi has a way todo this,so I replaced this with pixi's approach